examples/platforms/nrf52840/platform-config.h

```diff
@@ -48,6 +48,16 @@
  * @section UART Driver Configuration.
  ******************************************************************************/
 
+/**
+ * @def ENABLE_FEM
+ *
+ * Enable front end module
+ *
+ */
+#ifndef ENABLE_FEM
+#define ENABLE_FEM 0
```

Comment:
This define name and descirption may be a little misleading since FEM is built into our radio driver by default (and can be enabled/configured run time). What you want here, is to configure FEM to work with default GPIOs and PPIs.

Can we rename it to e.g. `PLATFORM_FEM_SET_DEFAULT_CONFIG` or `PLATFORM_FEM_ENABLE_WITH_DEFAULT_CONFIG` with proper comment.
It would be also nice to move this define closer to radio driver section couple lines below.
I would suggest to create a new section like @section Platform Fem configuration.

Thoughts?